protos/feast/core/FeatureService.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package feast.core;
+
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+option java_outer_classname = "FeatureServiceProto";
+option java_package = "feast.proto.core";
+
+import "google/protobuf/timestamp.proto";
+import "feast/core/FeatureReference.proto";
+
+message FeatureService {
+  // User-specified specifications of this feature service.
+  FeatureServiceSpec spec = 1;
+
+  // System-populated metadata for this feature service.
+  FeatureServiceMeta meta = 2;
+}
+
+message FeatureServiceSpec {
+  // Name of the Feature Service. Must be unique. Not updated.
+  string name = 1;
+
+  // Name of Feast project that this Feature Service belongs to.
+  string project = 2;
+
+  // List names of entities to associate with the Features defined in this
+  // Feature Service. Not updatable.
+  repeated string entities = 3;
```

Comment:
Is there a reason why entities are defined here? Aren't they already within the feature views?

Reply:
Good point, I don't think this is needed here.